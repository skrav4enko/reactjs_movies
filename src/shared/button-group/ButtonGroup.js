import React from 'react';
import './ButtonGroup.scss';

function ButtonGroup(props) {
  const { children } = props;

  return <div className="button-group">{children}</div>;
}

export default ButtonGroup;
