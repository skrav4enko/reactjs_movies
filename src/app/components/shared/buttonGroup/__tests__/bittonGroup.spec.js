/* eslint-disable */
import React from 'react';
import { shallow } from 'enzyme';
import ButtonGroup from '..';

describe('Button Group', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<ButtonGroup>children</ButtonGroup>);
    expect(wrapper).toMatchSnapshot();
  });
});
