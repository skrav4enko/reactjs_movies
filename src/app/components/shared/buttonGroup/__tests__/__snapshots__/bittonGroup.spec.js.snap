// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Group should match snapshot 1`] = `
<index.tsx>
  children
</index.tsx>
`;
